### PR TITLE
add new tcf fields to system model

### LINKED ISSUE #172: Add some missing TCF-related fields to `System` model
### Is your feature request related to a specific problem?

We need to add these fields to the `System` model:

- `cookie_max_age_seconds`: `Optional[int]`
- `uses_cookies`: `bool`, `default=False`
- `cookie_refresh`: `bool`, `default=False`
- `uses_non_cookie_access`: `bool`, `default=False`
- `legitimate_interest_disclosure_url`: `Optional[AnyUrl]`

### Describe the solution you'd like

Update the `System` model, shouldn't be more complexity than that

### Describe alternatives you've considered, if any

n/a

### Additional context

broader ticket covering the updates across our stack: https://ethyca.atlassian.net/browse/PROD-1147

### PR DESCRIPTION
**Pull request by adamsachs**
Closes #172 

### Description Of Changes

See issue description

### Code Changes

* [x] update `System` model
* [x] assert fields can be instantiated via automated tests

### Steps to Confirm

n/a, automated test coverage should be good enough for us here as this is a pretty straightforward additive change

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* [ ] Documentation Updated
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`

### CODE REVIEW

File: src/fideslang/models.py [hunk 1]
```diff
@@ -1138,10 +1138,10 @@ class System(FidesModel):
         description="Deprecated. The responsibility or role over the system that processes personal data",
     )
     egress: Optional[List[DataFlow]] = Field(
-        description="The resources to which the System sends data."
+        description="The resources to which the system sends data."
```
Comment (adamsachs) [inline src/fideslang/models.py:1141]: decided to standardize capitalization of `system` across the descriptions in this model class while i was here

File: src/fideslang/models.py [hunk 2]
```diff
@@ -1225,6 +1225,24 @@ class System(FidesModel):
     data_security_practices: Optional[str] = Field(
         description="The data security practices employed by this system."
     )
+    cookie_max_age_seconds: Optional[int] = Field(
+        description="The maximum storage duration, in seconds, for cookies used by this system."
```
Comment (adamsachs) [inline src/fideslang/models.py:1229]: Descriptions were informed by official spec's descriptions [here](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md#usescookies)

**Comment by adamsachs** [comment]
@pattisdr i think this should be a quick one, mind just sanity checking me that i haven't missed something silly?

@rsilvery i'd like a quick eye check on the descriptions i added for the new fields from a product POV please, thank you!

**Review by pattisdr — APPROVED** [review]
looks good

**Pull request merged**